src/application.ts

```diff
@@ -1,5 +1,6 @@
 import * as express from 'express'
 import {EventEmitter} from 'promise-events'
+import {Plugin} from '.'
```

Comment:
I'd love to rename this, but don't have any great ideas. `ApplicationCallback`?

Reply:
Definitely agree, but I think that `Callback` isn't quite right. Some maybe options:

* `ApplicationFunction`
* `AppFunction`
* `AppHandler`